examples/data-sources/b1ddi_dns_auth_nsgs/data-source.tf

```diff
@@ -16,3 +16,10 @@ data "b1ddi_dns_auth_nsgs" "tf_example_auth_nsg" {
 
 # Get all DNS Auth NSGs
 data "b1ddi_dns_auth_nsgs" "all_dns_auth_nsgs" {}
+
+# Get DNS Auth NSG with the specified tags
+data "b1ddi_dns_views" "all_dns_views_with_tags" {
+       tfilters = {
+            TestType  = "Acceptance"
```

Comment:
It seems the indentation not being consistent with previous blocks. Maybe in the future we should look at using https://developer.hashicorp.com/terraform/cli/commands/fmt to format all these files.

Reply:
@akleinib We will be adding linters in the new repo. Go and terraform linters will be mandatory checks for every PR.

Reply:
Fixed